ResourceStatusSystem/Command/FreeDiskSpaceCommand.py

```
@@ -95,15 +95,34 @@ def doNew( self, masterParams = None ):
     self.rpc = RPCClient( elementURL, timeout=120 )
     free = self.rpc.getFreeDiskSpace("/")
     total = self.rpc.getTotalDiskSpace("/")
-    self.rsClient.addOrModifySpaceTokenOccupancyCache(endpoint = elementURL, lastCheckTime = datetime.utcnow(),
-                                                      free = free, total = total, token = element )
+    result = self.rsClient.addOrModifySpaceTokenOccupancyCache(endpoint = elementURL, lastCheckTime = datetime.utcnow(),
+                                                               free = free, total = total, token = elementName )
+    if not result[ 'OK' ]:
+      return S_ERROR( DErrno.EMYSQL, "Query failed %s" % result )
```

no,
```
if not result['OK']:
    return result
```
Let the error propagate